app/config.js

```diff
@@ -35,8 +36,18 @@ const _watch = function() {
         onChange();
       }
     });
-  } else {
-    _watcher = fs.watch(cfgPath);
+    return;
+  }
+  // macOS/Linux
+  setWatcher();
+  function setWatcher() {
+    _watcher = fs.watch(cfgPath, eventType => {
+      if (eventType === 'rename') {
+        _watcher.close();
+        // Ensure that new file has been written
+        setTimeout(() => setWatcher(), 500);
```

Comment:
What would be our retry strategy? Only one time? Retry until it works?
If we think that 500ms is not enough, we need to put some try/catch too.

But I really think that it would work 99.99%. If not, maybe we'll add some retries later?

Mmmmh I should try this with config file remove...

Do not merge it please, I think it needs some more tests on some edge cases